websites/A/Armorgames/presence.ts

```diff
@@ -2,135 +2,87 @@ const presence = new Presence({
 		clientId: "827910536049852488",
 	}),
 	browsingTimestamp = Math.floor(Date.now() / 1000);
-let search: HTMLInputElement, title: HTMLElement;
+let search: HTMLInputElement;
 
 presence.on("UpdateData", async () => {
 	const presenceData: PresenceData = {
-			largeImageKey: "logo",
+			largeImageKey: "https://i.imgur.com/ofYOGAJ.png",
 			startTimestamp: browsingTimestamp,
 		},
-		page = window.location.pathname;
-	if (page === "/") {
-		search = document.querySelector("#search-input");
-		if (!search || search.textContent === "")
-			presenceData.details = "Viewing The Homepage";
-		else {
-			presenceData.details = "Searching:";
-			presenceData.state = search.value;
-			presenceData.smallImageKey = "searching";
-		}
-	} else if (page.includes("/category/")) {
-		if (page === "/category/all")
+		{ pathname, href } = window.location,
+		buttons = await presence.getSetting("buttons");
+	search = document.querySelector<HTMLInputElement>('[id="search-input"]');
+	if (search?.value || pathname.includes("/search")) {
+		presenceData.details = "Searching for";
+		if (
+			document.querySelector<HTMLInputElement>('[class="input-xlarge"]')?.value
+		) {
+			presenceData.state = document.querySelector<HTMLInputElement>(
+				'[class="input-xlarge"]'
+			).value;
+			presenceData.buttons = [
+				{
+					label: "Browse Searches",
+					url: href,
+				},
+			];
+		} else if (search?.value) presenceData.state = search?.value;
+		else delete presenceData.details;
+
+		presenceData.smallImageKey = "searching";
+	} else if (pathname === "/") presenceData.details = "Viewing The Home page";
```

Comment:
```suggestion
	} else if (pathname === "/") presenceData.details = "Viewing he home page";
```

Reply:
Looks like you've edited to fast, a `t` is missing for the `the` !